gulpfile.js

```diff
@@ -733,6 +540,169 @@ const copyRootTSConfig = ((cache) =>
+
+    if (fs.lstatSync(newInputPath).isDirectory()) {
+      copyBrowserifyTypesFiles(
+        newInputPath,
+        newInputPath.includes('@neo-one/ec-key') ? `${newOutputPath}-browserify` : newOutputPath,
+      );
+    } else if (dirOrFile.endsWith('.d.ts')) {
+      const typesFile = fs.readFileSync(newInputPath, 'utf-8');
+      fs.outputFileSync(
+        newOutputPath,
+        newOutputPath.includes('neo-one-types-browserify/index.d.ts')
+          ? typesFile.replace('ec-key', 'ec-key-browserify')
+          : typesFile.replace(/(?<='@neo-one\/)(.*)(?=')/gm, (match) => `${match}-browserify`),
+      );
+    } else if (dirOrFile === 'package.json') {
+      const pkgJSON = fs.readJSONSync(newInputPath);
+      fs.outputJSONSync(
+        newOutputPath,
+        {
+          ...pkgJSON,
+          name: `${pkgJSON.name}-browserify`,
+          dependencies:
+            pkgJSON.dependencies === undefined
+              ? undefined
+              : Object.entries(pkgJSON.dependencies).reduce((acc, dependency) => {
+                  const dependencyName = dependency[0].includes('@neo-one/')
+                    ? `${dependency[0]}-browserify`
+                    : dependency[0];
+                  const dependencyVersion = dependency[1];
+                  return {
+                    ...acc,
+                    [dependencyName]: dependencyVersion,
+                  };
+                }, {}),
+        },
+        { spaces: 2 },
+      );
+    } else if (dirOrFile.endsWith('.md')) {
+      fs.copySync(newInputPath, newOutputPath);
+    }
+  });
+};
+
+const bundleBrowserify = ((cache) =>
+  memoizeTask(cache, async function bundleBrowserify(format) {
+    if (format.dist === 'neo-one') {
+      const pkgJSONMap = createPkgJSONMap();
+
+      Object.entries(CLIENT_BROWSERIFY).forEach(([pkgName, neoOneDependencies]) => {
+        const externals = getClientPkgDependencies(pkgJSONMap, neoOneDependencies);
+        const basePath = path.resolve(__dirname, getDest(format));
+        const inputPath = path.resolve(basePath, pkgName);
+        const outputPath = path.resolve(basePath, `${pkgName}-browserify`);
+
+        webpack(webpackBrowserifyConfig(externals, inputPath, outputPath), (err, stats) => {
+          if (err) {
+            throw err;
+          } else if (stats.hasErrors()) {
+            console.log(
+              stats.toString({
+                performance: false,
+                hash: false,
+                timings: true,
+                entrypoints: false,
+                chunkOrigins: false,
+                chunkModules: false,
+                colors: true,
+              }),
+            );
+            throw new Error('Webpack bundling failed.');
+          }
+
+          neoOneDependencies.forEach((dependency) => {
+            const dirName = getDirName(dependency);
+            copyBrowserifyTypesFiles(path.resolve(basePath, dirName), path.resolve(basePath, `${dirName}-browserify`));
```

Comment:
I think you need to copy in the LICENSE and README.md files from the root of the project as well.

Reply:
Oh, I see, maybe you already are? Is there a LICENSE and README.md file in the output?